packs/journals/archetypes.json

```diff
@@ -988,7 +988,7 @@
ning as a ghoul, an undead cursed with a never-ending hunger for dead flesh that slowly overrides your reason and willpower until you would do anything for the meal you crave. In addition to the personality changes, your body changes as well. Your ears become pointed, and your skin grows blueish and pale. Your fingernails sharpen into claws; your teeth become razor sharp, ready to tear flesh off the bone; and your tongue grows long and pointed.</p><p>You gain the ghoul and undead traits and the basic undead benefits. Your undead craving is for the flesh of dead creatures; freshly dead corpses from sapient creatures fulfill your cravings much more than non-sapient or rotted corpses, but any dead creature can stave off the hunger for at least a little while. You gain two unarmed attacks in the brawling weapon group. The first unarmed attack is a claw that deals 1d4 slashing damage with the agile and finesse traits, and the second unarmed attack is a set of jaws that deal 1d6 piercing damage and have the finesse trait.</p><p>You gain the Consume Flesh action (see below). Satisfying your craving is difficult. Consuming Flesh temporarily satiates you. Many ghoul feats have extra abilities or functions that can be used when you are satiated, though using those abilities also causes you to lose your satiated status. You can always Consume Flesh, but eating more than a normal amount has no further effect; a ghoul's hunger is satiated, or it is not.</p><h3>Consume Flesh <span class=\"pf2-icon\">1</span> (manipulate)</h3><p>@Embed[Compendium.pf2e.actionspf2e.Item.1If9lLVoZdO8woVg inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Replenishing Consumption] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.NQtIhIowH1tVywZI inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Swift Leap] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.VP2CWUTZ9Edg82uz inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Feverish Enzymes] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.c5SfaSn6OEHkHxII inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Grave Strength] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.7GrACprIxZuarGDs inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Paralyzing Slash] <span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.8z9XkfZalQ5tUjfy inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Sickening Bite] <span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.Aww98EQXcigRhY3v inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Glutton for Flesh] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.5GC2iGtVp3UAH2nm inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Corpse Stench] <span style=\"float:right\">Feat 12</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.U5hjS5qFrtGHWlVG inline]</p>",
```

Comment:
```suggestion
                "content": "<p>You died from the necrotic disease known as ghoul fever and rose anew as a ravenous ghoul, forever craving the flesh of sapient creatures. While you can temper your hunger with long-dead flesh stolen from ancient graves, you much prefer devouring those who are freshly dead, whether as a scavenger or because you killed the creature yourself to ensure a fresh meal.</p><p>Whether they stalk through the graveyards and crypts of Golarion or work together with other ghouls to build empires in the Darklands, ghouls care most about the consumption of dead flesh. The moment you become a ghoul, your existence focuses upon satisfying this endless craving. Ghouls who go more than a few days without feeding on freshly dead humanoid flesh find themselves in constant pain; those who go for more than a week might be driven to the edge of delirium, becoming bestial in their craving.</p><p>When you succumb to ghoul fever and become a ghoul, you don't lose your mind or memories. You keep your personality only to have it warped by the never-ending need to consume flesh. A brave warrior remains brave, but under the curse of the fever, that brave warrior's prime directive is to consume flesh. A rogue might go from finding clever ways to pick pockets to finding ways to waylay opponents and devour them.</p><p>Since most societies don't allow cannibalism, ghouls who wish to continue their old lives must satisfy their cravings in secret, finding a way to balance society's rules while feeding the constant gnawing hunger inside them. Some take up professions that allow them to secretly eat dead flesh or create dead bodies to feed upon. There are tales told of ghoul mercenary troops who satisfy their cravings and profit, while ghoul spellcasters can use their magic to procure and preserve flesh.</p><p>Ghouls who won't restrain themselves with rules of the living may join or establish ghoul societies. These caste systems base their hierarchy on how much flesh one has consumed and can provide for other ghouls. Hunger and ambition drive ghouls to rise through the ranks, which allows them to feed their hunger even more.</p><p><strong>Additional Feats</strong>: 6th @UUID[Compendium.pf2e.feats-srd.Item.Guarded Movement], @UUID[Compendium.pf2e.feats-srd.Item.Reactive Pursuit]; 10th @UUID[Compendium.pf2e.feats-srd.Item.Wall Run]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Ghoul Dedication] <span style=\"float:right\">Feat 2</span></h2><p><strong>Prerequisites</strong> You were killed by ghoul fever.</p><hr /><p>You have succumbed to ghoul fever, dying and returning as a ghoul, an undead cursed with a never-ending hunger for dead flesh that slowly overrides your reason and willpower until you would do anything for the meal you crave. In addition to the personality changes, your body changes as well. Your ears become pointed, and your skin grows blueish and pale. Your fingernails sharpen into claws; your teeth become razor sharp, ready to tear flesh off the bone; and your tongue grows long and pointed.</p><p>You gain the ghoul and undead traits and the @UUID[Compendium.pf2e.ancestryfeatures.Item.Basic Undead Benefits]. Your undead craving is for the flesh of dead creatures; freshly dead corpses from sapient creatures fulfill your cravings much more than non-sapient or rotted corpses, but any dead creature can stave off the hunger for at least a little while. You gain two unarmed attacks in the brawling weapon group. The first unarmed attack is a claw that deals 1d4 slashing damage with the agile and finesse traits, and the second unarmed attack is a set of jaws that deal 1d6 piercing damage and have the finesse trait.</p><p>You gain the Consume Flesh action (see below). Satisfying your craving is difficult. Consuming Flesh temporarily satiates you. Many ghoul feats have extra abilities or functions that can be used when you are satiated, though using those abilities also causes you to lose your satiated status. You can always Consume Flesh, but eating more than a normal amount has no further effect; a ghoul's hunger is satiated, or it is not.</p><h3>Consume Flesh <span class=\"pf2-icon\">1</span> (manipulate)</h3><p>@Embed[Compendium.pf2e.actionspf2e.Item.1If9lLVoZdO8woVg inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Replenishing Consumption] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.NQtIhIowH1tVywZI inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Swift Leap] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.VP2CWUTZ9Edg82uz inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Feverish Enzymes] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.c5SfaSn6OEHkHxII inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Grave Strength] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.7GrACprIxZuarGDs inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Paralyzing Slash] <span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.8z9XkfZalQ5tUjfy inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Sickening Bite] <span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.Aww98EQXcigRhY3v inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Glutton for Flesh] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.5GC2iGtVp3UAH2nm inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Corpse Stench] <span style=\"float:right\">Feat 12</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.U5hjS5qFrtGHWlVG inline]</p>",
```